docs/img/EventNotificationSequence.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="app.diagrams.net" modified="2020-08-18T11:27:01.337Z" agent="5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.125 Safari/537.36" etag="rIwX5N_5Yl_RqGqX_Tgy" version="13.4.9" type="device"><diagram id="kgpKYQtTHZ0yAKxKKP6v" name="Page-1">5VrbUuM4EP2aVO0+kLIlO5fHJDAMVcxALVMF7Jtiy7YWxTKynAtfv5ItX5WEDBCYYVI8RC2pLfdpnT5S6MHZYn3OURJ9Yz6mPWD56x487QFgOwD01J/lbwrL0IaFIeTE14Nqww15wtpoaWtGfJy2BgrGqCBJ2+ixOMaeaNkQ52zVHhYw2n5qgkJsGG48RE3rLfFFVFhHrlXbv2ISRuWTbUv3LFA5WBvSCPls1TDBsx6cccZE8W2xnmGqglfGpZj3ZUdvtTCOY3HIBBhn0y9rPvY2l1YSX93+AOj8xC28LBHN9At/zeZ6vWJTBkEuPVFfswW9JAGmJJataYI5WWCBueyh2nxd26YSEoGkTfXbeZtSlKRknru1pIVjL+MpWeJ/cFogn1tZFvvY17OqsKmuVHD2UAGhus0o6MAsMRd43TDpqJxjJhfIN3KI7gUlmDpFR7q5qvGuUI0aWMPSiHSOhZXrGgb5RSPxE6gMDFQMSBJGYpE/15323NMOHIyLiIUsRrQJSB1Y6yWB3ZtAB0fbcZ8Pthlre3CsWI+MWJ8t1SsD6zsTJCAeEoTFRvzVC8s+OqEkjKVpzoRgCxk3HPsTxTvKRpn38IoUxn6Lm8yQckzl6pZtqtoWIT31WiVNDQW02ljYI9h2kbKMe1jPalJLx5HjOv1nXAnEQywMVzlk1RsdhKJ/cW/Ru3+nNxdeehp5364n49WJbaB4i+cRYw/SOEmSHpBbCtoq/n+l2Tz1OJlj/vcfRnSOfQDRgSMR3VbYwC9JdHsz7HcgOpBeLNf3T+vvY+spQEFI6C08pKiUpHaJ5phes5Tk1Ncgty7rCZZIayQWtE7ifAMt1qHSg/05SonXDyhKIyPhA7lttODLd+aOAB+KzU4gBu2st10TCbC1vB8JiKEBxEUs81bmcKFj5esrbvUiFEv+72KU97Zjnqf0jFGmdkTMcs4KCKUdE9KgeTgutkkXzQXxffWY6SoiAt8kyFPPXEkgjT31XtgNO9ht2UXOFuzAsQjLVAu5XrbiXCvIg4rEStcXEodqBUlCtYaQnTNV1YmIZPilE4LUmxR1Ss1kvGinUjZgyXm7qpN82C6UGmkxR95DmKN2lYmikuV2H/GHKzmLCBVUqy9FQHdnlplCcaByTZcse/g2ZWgAuxvSNqlx244cjI9VhhyTG90pUGXPkhLCPT2m/GtuJTv3IF/pTnfmjfscJwe6peF0rb0VrU2z1aiDhXFNxF0B9Gig2y2Hsl37U41No9H15ksir9Lkw4Sr47r9kTuuP618AnDQH1iND3iZqu3K4+f8vp3E3Vo2TIlbUA9Ja/7xVcIGak2tUtKQv2VVsUisxsa4nCFJqT2y0sm+QWPvtxm25tOWnHvzFLNLTihJqks+h2YRGMPu2ajr6siJAz+F8NtfU6p6UcbYPkzkOXsS41Uiz6wot2Vh/2RH0lfh5jofePe2FTjzRtSA5J2OpM8Ednfa/TZH0o+/e6vEVqWfCm0EbfgSsXVouZLlNuf85/JwJ5InUr4NdfhefQs4aMscu+Pi0Ernjqz9jnbUudpROZAFQYqPUgvH5u4eThtU3JuB3sTSXiaFIHrM1E8008fxEv6XzX3/UQxgbNcd+bHO9JAKJLK06yU/VNQz62nD9z1kHO9U0Dk8vlzB6bSy+iMAhm+S6u7I3nNugLb1ssSXB4WOxDNc/QKpXy7pU+vAQeeKG4w/+LLPNo9tf8ht388BBcHYAOqNbvZks/6VvdhN9f8qwLP/AQ==</diagram></mxfile>
```

Comment:
Would it be worth having one of the apps respond instead with a 202 to show that received the notification but need more time to actually perform the change? I think this would basically mean moving the "internal context change" lower and maybe a waiting period (arrow that loops back itself?) saying "validating event notification" or something similar.

Reply:
Good idea, diagram updated